nav2_regulated_pure_pursuit_controller/README.md

```diff
@@ -117,6 +119,8 @@ controller_server:
       use_cost_regulated_linear_velocity_scaling: false
       regulated_linear_scaling_min_radius: 0.9
       regulated_linear_scaling_min_speed: 0.25
+      use_fixed_curvature_lookahead: true
```

Comment:
Default to false -- mostly since the "usual" user of Nav2 isn't going to be running a tractor :laughing:

Reply:
Ok will change (the default is actually false, but I will match here)

Reply:
Done